slack/apps-and-integrations/apps-and-integrations.md

```diff
@@ -0,0 +1,12 @@
+# List of apps and integrations used
+
+## Productboard
+- **Description:** Productboard is a product management tool, used by the PrestaShop Core's product team to gather feedback, organize and pritoritize features and improvements to build, and to follow their development.
+The Productboard app connects this Slack to the PrestaShop's Productboard.  
+It allows the product managers to transform a Slack message (and its comments) into a feedback "note" on productbaord.  
+More info on the productboard website: https://www.productboard.com/
+- **Access:** Limited
+- **Users:** Only the PrestaShop's productboard users can create note from Slack
+- **Owners:** PrestaShop Core's product management team, coreproduct@prestashop.com
+- **Language:** EN 
+- **Channels perimeter:** all channels
```

Comment:
all public channels ? or all channels where one of your team has an access ?

Reply:
Well, all channels, but as I restricted the use to only PB users (on PB side) only a PM can collect a feedback. Thus, a PM needs to be in the channel to collect the feedback, but the app can be used in any channel (private and public)